Modules/mmapmodule.c

```diff
@@ -1272,7 +1272,7 @@ new_mmap_object(PyTypeObject *type, PyObject *args, PyObject *kwdict)
     DWORD off_lo;       /* lower 32 bits of offset */
     DWORD size_hi;      /* upper 32 bits of size */
     DWORD size_lo;      /* lower 32 bits of size */
-    const char *tagname = "";
+    PyObject *tagname = Py_None;
```

Comment:
This should be `NULL`, not an uncounted reference to `None`.

Reply:
No, it should be `Py_None` (please see my other comment). Also, for what it's worth, using an uncounted reference to `Py_None` in this way is safe (it's also done in many places in the interpreter core).